internal/vrf/testKeyTransceiver.go

```diff
@@ -30,7 +30,7 @@ func (kt *keyTransceiver) KeyLookup(p contract.KeyID) dkg.KeyData {
 		return *kt.kd.Clone()
 	}
 	if kt.kd == nil {
-		panic("key consumer is asking for unavailable key")
+		return dkg.KeyData{}
```

Comment:
Hmm.. this file shouldn't get exported. We'll have to fix the export script later.